developer_guides/debugability/logger/index.rst

```diff
@@ -124,3 +125,70 @@ Examples:
   .. code-block:: bash
 
      sof-logger -l ldc_file -d
+
+
+Trace filtering
+***************
+
+It's possible to change current logs level for any instance of component in
+runtime. To do so, use `-F` option with following argument format:
+
+   <log_level>="<component1>[, <component2>]"
+
+Where *<log_level>* is one of:
+
+- ``c`` / ``critical``
+- ``e`` / ``error``
+- ``w`` / ``warning``
+- ``i`` / ``info``
+- ``d`` / ``debug``
+- ``v`` / ``verbose``
+
+After **=** character, there is a list of component, separated with comma.
+Each components starts with *name* followed by optional part with *instance*
+description.
+
+List of possible components names comes from UUID declaration,
+see :ref:`uuid-api` for more detailed information.
+Try use ``-d`` flag in logger to list component names from `ldc` file content.
+There is also a special name - ``*`` - used to apply given trace
```

Comment:
Shall we use the term "wildcard" for * .

Reply:
changed